cdi-unit/src/main/java/org/jglue/cdiunit/internal/TestConfiguration.java

```diff
@@ -21,7 +21,7 @@ public TestConfiguration(Class<?> testClass, Method testMethod, Collection<Class
     }
 
     public TestConfiguration(Class<?> testClass, Method testMethod) {
-        this(testClass, testMethod, new ArrayList<>());
+        this(testClass, testMethod, Collections.EMPTY_SET);
```

Comment:
👍 If you want to go one better, use Collections.emptyList(), which is type safe.